nfs-ganesha/build/build_deb

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/bash
+set -ex
+
+# Only do actual work when we are a DEB distro
+if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
+    exit 0
+fi
+
+
+## Install any setup-time deps
+# TODO -- upload install-deps.sh script that installs the setup time deps
+# to upstream repo or update this section to install these packages
+
+# We need this for mk-build-deps
+sudo apt-get install -y equivs cmake git
+
+## Setup the pbuilder
+# TODO remove if you do not want to use pbuilders
+setup_pbuilder
+
+
+## Get some basic information about the system and the repository
+# TODO -- update this to get the proper VERSION/REVISION
+DEB_ARCH=$(dpkg-architecture -qDEB_BUILD_ARCH)
+# ALI TO DO GET VERSION
+VERSION="$(./get-version.sh)"
+
+
+## Build the source tarball
+# ALI TO DO: find out if this can be set in config/definitions/nfs-ganesha.yaml
+NFS_GANESHA_BRANCH="next"
```

Comment:
yes, you can specify what branch(es) you want to build in the configuration. See http://docs.openstack.org/infra/jenkins-job-builder/scm.html#scm.git